opencodelists/tests/management/commands/test_setup_local_dev_databases.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from django.core.management import call_command
+
+from opencodelists.management.commands import setup_local_dev_databases
+
+
+def test_setup_local_dev_databases_exits_early_when_in_prod_environment(monkeypatch):
+    monkeypatch.setattr(setup_local_dev_databases, "in_production", lambda: True)
```

Comment:
A purist might say this doesn't quite test whether this would run in production or not. If there were some bug in `in_production()`'s interpretation of django settings then this test could give an erroneously false pass